src/screens/ImportSeedPhraseSheet.js

```diff
@@ -105,34 +110,48 @@ const ImportButton = ({ disabled, onPress, seedPhrase }) => (
 );
 
 const ImportSeedPhraseSheet = ({ isEmpty, setAppearListener }) => {
+  const { accountAddress } = useAccountData();
   const { clipboard } = useClipboard();
   const { navigate, setParams } = useNavigation();
   const initializeWallet = useInitializeWallet();
   const [isImporting, setImporting] = useState(false);
   const [seedPhrase, setSeedPhrase] = useState('');
+  const [startFocusTimeout] = useTimeout();
+  const [startAnalyticsTimeout] = useTimeout();
+
+  const isClipboardValidSeedPhrase = useMemo(() => {
+    return clipboard !== accountAddress && validateSeed(clipboard);
+  }, [accountAddress, clipboard]);
+
+  const isSeedPhraseValid = useMemo(() => {
```

Comment:
Since we also accept pkeys, what about calling these two fn `isClipboardValidSecret` and `isSecretValid`